### PR TITLE
[MM-37725] - Self-managed: Update email template to new branding - Password Reset

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR updates the password reset email template to new branding

Screenshot 2021-08-13 at 13 59 25

#### Ticket Link
[MM-37725](https://mattermost.atlassian.net/browse/MM-37725)

#### Release Note
```release-note
NONE

```

### CODE REVIEW

**Review by neallred — COMMENTED** [review]
Code looks good to me aside from the comment in the [webapp side](https://github.com/mattermost/mattermost-webapp/pull/8582/files) requesting to remove the old picture.

**Comment by stevemudie** [comment]
@AGMETEOR The email confirmation (when I signed up as a new user) contained the updated graphics but the password reset one didn't have any graphic in it.

Screen Shot 2021-08-13 at 1 24 20 PM

**Comment by AGMETEOR** [comment]
@stevemudie this PR has a webapp part here https://github.com/mattermost/mattermost-webapp/pull/8582 where the image lives. The test server is not picking up the webapp code. We can test this post merge?

File: templates/reset_body.html [hunk 1]
```diff
@@ -106,17 +106,21 @@
...
       letter-spacing: -0.01em !important;
-      color: #3D3C40 !important;
+      color: #3F4350 !important;
     }
 
     .subTitle div {
       font-size: 16px !important;
       line-height: 24px !important;
-      color: rgba(61, 60, 64, 0.64) !important;
+      color: rgba(63, 67, 80, 1) !important;
```
Comment (pablovelezvidal) [inline templates/reset_body.html:115]: when opacity value is 100% (1), the alpha channel is not needed, so no need to use the four channels rgba function. The rgb(r, g, b) is enough.

**Review by pablovelezvidal — APPROVED** [review]
Besides the comment about the RGBA function, everything else looks good to me.

**Review by stevemudie — APPROVED** [review]
Will tests post merge due to the image not being picked up here.

**Review by neallred — APPROVED** [review]
LGTM. Thanks, @AGMETEOR !

**Review by andrewbrown00 — CHANGES_REQUESTED** [review]
Thanks, @AGMETEOR, one item to resolve:

- [ ] We're missing the **Questions?** section below the link expires language

Current | Design
----- | -----
image | image


[Here's a link to the design file for reference](https://www.figma.com/file/TnHl6HeN0Z9n7zYXx6hS5T/MM-37612-Update-Emails-to-align-with-new-brand?node-id=522%3A60939).

**Comment by AGMETEOR** [comment]
> Thanks, @AGMETEOR, one item to resolve:
> 
> * [ ]  We're missing the **Questions?** section below the link expires language
> 
> Current	Design
> image	image
> [Here's a link to the design file for reference](https://www.figma.com/file/TnHl6HeN0Z9n7zYXx6hS5T/MM-37612-Update-Emails-to-align-with-new-brand?node-id=522%3A60939).

@andrewbrown00 that’s how it works when the support email is not set from the admin console. When the support email is set, the questions section will show

**Comment by andrewbrown00** [comment]
> @andrewbrown00 that’s how it works when the support email is not set from the admin console. When the support email is set, the questions section will show

Thanks, @AGMETEOR, for providing that context. I wasn't aware of that. We could probably be consistent across the emails as a way for users to have an option to always reach us. We can revisit the UX to review and make a decision, create a separate ticket if needed.

**Comment by AGMETEOR** [comment]
> > @andrewbrown00 that’s how it works when the support email is not set from the admin console. When the support email is set, the questions section will show
> 
> Thanks, @AGMETEOR, for providing that context. I wasn't aware of that. We could probably be consistent across the emails as a way for users to have an option to always reach us. We can revisit the UX to review and make a decision, create a separate ticket if needed.

We can maybe create a separate ticket

**Review by andrewbrown00 — APPROVED** [review]
Thanks, @AGMETEOR, changes look good 🙂 , based on the previous comment about the generic reset vs being sent from support.

image

**Pull request merged**